dartboard/DartboardView.py

```diff
@@ -0,0 +1,299 @@
+
+         # this call could be used to get your mock points
+         print(event.pos())
+
+         #
+         hit = DartboardHit(x=event.pos().x(), y=event.pos().y())
+         hit.save()
+         print(DartboardHit.objects.all())
+
+         #this stores the real position of the click by adding together the top left position of the window with the click position relative to the window
+         realPosition = QPoint(event.pos().x() + self.WindowPosX, event.pos().y() + self.WindowPosY)
+
+         #check for collisions with each region
+         if (self.bullseye.containsPoint(realPosition, Qt.OddEvenFill)):
+             print("bullseye")
+         #elif (self.outer_bullseye.containsPoint(realPosition, Qt.OddEvenFill)):
+             #print("outer bullseye")
+
+         #this line adds an entry for the point that was just placed. It calculates the x and y distance from the center of the circle divided by the current radius of the circle
+         #this is used to relocate the points later when the window is resized
+         self.points.append([(event.pos().x() - (self.WindowSizeX / 2.0)) / self.radius, (event.pos().y() - (self.WindowSizeY / 2.0)) / self.radius])
+
+         #this line draws the point on the screen where the mouse was clicked
+         self.scene.addEllipse(self.WindowPosX + event.pos().x(), self.WindowPosY + event.pos().y(), 3,3, QPen(), QBrush(Qt.white))
+
+    #this function is called when the window is resized
+    def resizeEvent(self, event):
+        #print(self.sceneRect())
+
+        #these lines set the variables initialized in the constructor to their new values
+        self.WindowSizeX = event.size().width()
+        self.WindowSizeY = event.size().height()
+        self.WindowPosX = self.pos().x()
+        self.WindowPosY = self.pos().y()
+
+        #this line sets the Scene's geometry based on the newly resized window
+        self.scene.setSceneRect(self.WindowPosX, self.WindowPosY, self.WindowSizeX, self.WindowSizeY)
+
+        #recall the drawing function to replace the circles and points with the new size
+        self.DrawRegions()
+
+        #tbh idek what this does
+        super().resizeEvent(event)
+        
+    #helper used to calculate circular coordinates
+    def GetPointsInCircle(self,r, n=20):
+        n = self.regionVertices
+        pi = math.pi
+        points = [QPoint(math.cos(2*pi/n*x)*r + self.WindowPosX + (self.WindowSizeX / 2.0), math.sin(2*pi/n*x)*r + self.WindowPosY + (self.WindowSizeY / 2.0)) for x in range(0,n+1)]
+        for p in points:
+            self.rotate(p, -pi/19)
+        return points
+
+    def rotate(self, point, angle):
+        centerX = self.WindowPosX + (self.WindowSizeX / 2.0)
+        centerY = self.WindowPosY + (self.WindowSizeY / 2.0)
+        point.setX(centerX + math.cos(angle) * (point.x() - centerX) - math.sin(angle) * (point.y() - centerY))
+        point.setY(centerY + math.sin(angle) * (point.x() - centerX) + math.cos(angle) * (point.y() - centerY))
```

Comment:
Please update to something like this (performs an actual rotation of the points) (also note the update to `-pi/20`)

```suggestion
            self.rotate(p, -pi/20)
        return points

    def rotate(self, point, angle):
        # Get Center of Dartboard
        centerX = self.WindowPosX + (self.WindowSizeX / 2.0)
        centerY = self.WindowPosY + (self.WindowSizeY / 2.0)
        
        # Calculate important trig information
        sin = math.sin(angle)
        cos = math.cos(angle)
        
        # Offset points so the center is the origin
        px = point.x() - centerX
        py = point.y() - centerY
        
        # Calculate new rotated points
        xnew = (px * cos) - (py * sin)
        ynew = (px * sin) + (py * cos)
        
        # Update actual point data
        point.setX(xnew + centerX)
        point.setY(ynew + centerY)
```